.gitignore

```diff
@@ -48,3 +48,5 @@ Gemfile.lock
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+.tool-versions
```

Comment:
You shouldn't ignore this, to make sure everyone cloning uses the same versions 😄